Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /usr/share/kibana
 
 ENV PATH=/usr/share/kibana/bin:$PATH
 
-RUN kibana-plugin install "https://github.com/dlumbrer/bitergia-analytics-plugin/releases/download/0.0.1_7.10.1/bitergiaAnalytics-7.10.0.zip"
+RUN kibana-plugin install "https://github.com/Bitergia/bitergia-analytics-plugin/releases/download/0.0.1_7.10.0/bitergiaAnalytics-7.10.0.zip"
```

Comment:
The minor number of the version is `1` and `0` in the other. Is there any specific reason for it?

Reply:
That is for the Kibana version used for the plugin (in this case is 7.10.0 for Open Distro 1.12.0), I'm my personal repository release I put `7.10.1` by mistake, the right version number is `7.10.0`

Reply:
Understood. No changes are needed.